experimenter/experimenter/jetstream/tasks.py

```diff
@@ -16,14 +16,19 @@
 @metrics.timer_decorator("fetch_experiment_data")
 def fetch_experiment_data(experiment_id):
     metrics.incr("fetch_experiment_data.started")
+    experiment = None
     try:
         experiment = NimbusExperiment.objects.get(id=experiment_id)
         experiment.results_data = get_experiment_data(experiment)
         experiment.save()
         metrics.incr("fetch_experiment_data.completed")
     except Exception as e:
         metrics.incr("fetch_experiment_data.failed")
-        logger.info(f"Fetching experiment data for {experiment_id} failed: {e}")
+        failure_message = f"Fetching experiment data for {experiment_id} "
+        if experiment is not None and hasattr(experiment, "slug"):
```

Comment:
I think you can remove `is not None ` completely and can do just `if experiment and ....`

Reply:
I have been bitten by using `if x` instead of `if x is not None` in the past and always default to the latter. Is there a reason to use `if experiment` here (and _not_ `if experiment is not None`)?

Reply:
no major concerns, just a suggestion :)